src/app/states/editor/page.mdx

```diff
@@ -0,0 +1,64 @@
+export const metadata = {
+  title: 'editor',
+  description: 'An app state of the editor. This section explains how to fetch and manipulate the `editor` state.',
+}
+
+# editingNote
+
+An [app state](/guides/flux-architecture) of the editor.
+You can obtain the `editor` state and integrate it with React components.{{ className: 'lead' }}
+
+
+---
+
+## Data Structure
+
+<Row>
+  <Col>
+    <Properties>
+      <Property name="changed" type="boolean | undefined">
+        `true` if the note has been changed. 
+        The editor will save the note if `changed` is true when it is closed.
+        Otherwise it will be`undefined` if it has not been changed yet since the app launched.
+      </Property>
+      <Property name="needsUpdatingDate" type="boolean | undefined">
+        Whether the note's `updatedAt` field needs to be updated or not.
+      </Property>
+      <Property name="viewMode" type="EditorViewMode">
+        The editor view mode which represents either `'preview'`, `'sideBySide'` or `'edit'`.
+      </Property>
+      <Property name="openingNoteId" type="string | undefined">
+        When the app is trying to open a note, it indicates its' note ID, otherwise `undefined``.
```

Comment:
duplicate backticks on the 'undefined'.

<img width="109" alt="Screenshot 2023-09-05 at 10 51 41" src="https://github.com/inkdropapp/api-docs/assets/1332805/fe7b8808-b20a-40a8-955f-f33521796016">

Reply:
Fixed